src/Ninja/G1CP/Content/Fixes/Session/fix278_EN_WomanOuMismatch.d

```diff
@@ -0,0 +1,7 @@
+/*
+ * #278 Ambient women dialog subtitles don't match the audio (EN)
```

Comment:
"Ambient women SVM subtitles don't match the audio (EN)"

Reply:
Updated. (Removed the word "ambient", because it is redundant when referring to SVMs.